### PR TITLE
Clearml and Google Drive

### PR DESCRIPTION
**Pull request by johnml1135**
…other functions (preprocess, train and test).

### CODE REVIEW

**Comment by johnml1135** [comment]
We can merge now to try out - I have not tested other functions.  They will need to have clearml code added to them as well.  This could be done by anybody though and the lifting of getting S3 working is done.  I have S3 mounted on my PC, but we also need to work and get it mounted on the other drives as well.

There is a significant possibility that S3 will not like being treated as a folder structure (appends, other trickery) and may crash and need to be restarted.  If this becomes a bigger issue, we may need to investigate other solutions.

I documented my work trying to resolve this issue here: https://stackoverflow.com/questions/67496760/mounting-an-s3-bucket-in-docker-in-a-clearml-agent/67497264#67497264

**Comment by johnml1135** [comment]
This is a fix for #7 issue.

File: scripts/clearml_agent/mount_aqua_on_data.bat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+rem Download rclone from: https://rclone.org/
+rem Extract into a folder.
+rem Copy this file and rclone.conf to the new folder
+rem You will also need the api key and secret installed through awscli for this to work.
+rem Note that you can map \\wsl$\Ubuntu to a drive letter and then browse the data.
+call wsl.exe fusermount -uz /data/aqua-ml-data
```
Comment (ddaspit) [inline scripts/clearml_agent/mount_aqua_on_data.bat:6]: What about Linux? Also, maybe this path should be specified by the `SIL_NLP_DATA_PATH` environment variable.

File: silnlp/common/environment.py [hunk 2]
```diff
@@ -12,6 +12,9 @@ def get_data_dir() -> Path:
     sil_nlp_data_path = os.getenv("SIL_NLP_DATA_PATH")
     if sil_nlp_data_path is not None:
         return Path(sil_nlp_data_path)
+    auqa_ml_path = Path("/data/aqua-ml-data")
```
Comment (ddaspit) [inline silnlp/common/environment.py:15]: rclone should be mounted to the path specified by `SIL_NLP_DATA_PATH`, then this code won't be necessary.

File: silnlp/common/extract_corpora.py [hunk 3]
```diff
@@ -26,6 +29,10 @@ def main() -> None:
 
     projects: Set[str] = set(args.projects)
 
+    task = Task.init(
```
Comment (ddaspit) [inline silnlp/common/extract_corpora.py:32]: I guess this is mostly done as a test ClearML task. I don't know how useful this would be in real-world use.

**Review by ddaspit — CHANGES_REQUESTED** [review]

**Comment by johnml1135** [comment]
Have ClearML be an option in Extract Corpora
Make a nmt/experiment.py file - base experiment (no clearml) class for preprocess, train and test.  Child class with ClearML.

**Comment by johnml1135** [comment]
Run with 3 steps (create checkpoint there) - without GPU.

**Comment by johnml1135** [comment]
Damien, are you OK merging this in now?

**Comment by johnml1135** [comment]
Needs one more commit after clearml 1.0.3 comes out to update the lock file.

**Comment by ddaspit** [comment]
I didn't get a chance to review this until today. The changes look good to me.

**Comment by ddaspit** [comment]
Feel free to delete the branch if you don't need it anymore.

**Pull request merged**